go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	github.com/spf13/cobra v1.4.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.7.1
+	github.com/vektah/gqlparser/v2 v2.4.2 // indirect
```

Comment:
This project seems to have some deps which themselves have deps. So we'll likely pull down a bunch of deps. Do we need to vendor the entire library or can we take the relevant parts and put them in `internal` ?

Reply:
I'm marking this as resolved, since I went with what @srenatus suggested.